src/conversion_helper.js

```diff
@@ -165,6 +165,18 @@ const unitLookupList = [
     "conversionFunction" : (i) => velocityMap(i * 0.3048) // 1 ft/s = 0.3048 m/s
   },
   {
+    "imperialUnits" : [/(?:pounds?|lbs?)\/(?:inch|in)/] ,
+    "standardInputUnit" : " lbs/inch",
+    "isInvalidInput" : isZeroOrNegative,
+    "isWeaklyInvalidInput" : isHyperbole,
+    "conversionFunction" : (i) => createMap(i * 0.017858, " kg/mm"),  // 1 lbs/inch = 0.017858 kg/mm
+    "ignoredUnits" : [/newton[ -]?met(?:er|re)s?/, /Nm/, /kg\/mm/],
+    "postprocessInput" : (input) => {
+      const nm = input * 0.0098; // 1 kg/mm = 0.0098 N/m
```

Comment:
Ah, this is not how we currently try to handle "or" cases for units. You can return an array of {number, unit} so you don't do anything in postprocessInput. :)
See [line 140](https://github.com/cannawen/metric_units_reddit_bot/blob/master/src/conversion_helper.js#L140) for example.